elements/pfe-icon/src/pfe-icon.scss

```diff
@@ -1,8 +1,22 @@
 @import "../../pfe-sass/pfe-sass";
-@include pfe-theme-contexts; // imports on="light" etc support
-
+@include pfe-theme-contexts;
```

Comment:
hey @castastrophe , on the [webux1](https://web-dev-drupal-webux1.dev.a1.vary.redhat.com/en/web-components-demo#) sandbox, the pfe-icon appears to not be working, is the latest pushed up to that box? 
![image](https://user-images.githubusercontent.com/456863/87046900-c3737d80-c1c7-11ea-96aa-7494d466485c.png)

Reply:
No I haven't pushed anything up to webux1 in several weeks.  I can rebuilt it if you like.